test/test-config.sh

```diff
@@ -48,6 +48,10 @@ TEST_PMEM_MEM_SIZE=32768 # 32GB
 TEST_PMEM_SHARE=on
 TEST_PMEM_LABEL_SIZE=2097152
 
+# Kubernetes feature gates to enable/disable
+# featurename=true,feature=false
+TEST_FEATURE_GATES="CSINodeInfo=true,CSIDriverRegistry=true"
```

Comment:
The installation of the corresponding CRDs also belongs into the cluster setup. Currently the PR has that as part of the driver installation (like in some older CSI documentation), but that's conceptually the wrong place and was just a workaround for cluster setup tools not knowing about those CRDs yet.

Reply:
agree. Done